### PR TITLE
docs: update changelog

### PR DESCRIPTION
**Pull request by ksted**
## Why:

Release v1.7.0

## Describe your changes

- Updates EN changelog
- Updates DE changelog

## Checklist before requesting a review

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: docs/changelog/changelog_en.md [hunk 1]
```diff
@@ -1,49 +1,46 @@
...
-- Fixed an accessibility issue where the font size was too small.
-- Fixed an issue where product path was not reactive when category was changed.
-- Automatically generate a language file for every active language, not just the default language.
-- Soft login was still shown after successfully authenticating on the order confirmation page.
-- Fixed an issue where manufacturer data was introducing 'name' into structured data instead of 'externalName'.
-- Headlines now use the configured font.
-- Fixed layout shift on category page.
-- The build script failed on Windows because of incompatibilities between file name pattern and operating system. The file name pattern now works on Windows.
+- Fixed an issue where increasing the quantity beyond maximum stock would clear the cart.
+- Removed the "Add to Cart" notification from item and category pages when the quick checkout modal is not present.
```
Comment (Tim-M-S) [inline docs/changelog/changelog_en.md:22]: Isn't that controlled by a setting?

File: docs/changelog/changelog_en.md [hunk 2]
```diff
@@ -1,49 +1,46 @@
...
 
-- Clicking "Buy" with an unsaved Address will now display a notification.
-- When an EU country is selected as the shipping country, all EU countries become available as billing options.
-- Added a new request header for configId and added no cache to environment variables.
-- Implement new notification design
-- Adding the ability to have alt text for images.
-- It's now possible to change the font color of the hero banner via a template property.
-- Display product titles without global suffix when available, maintaining backward compatibility.
+- Clicking the **Buy** button with an unsaved address now displays a notification.
+- Selecting an EU country as the shipping country now makes all EU countries available as billing options.
```
Comment (Tim-M-S) [inline docs/changelog/changelog_en.md]: This ensures compliance with geo-blocking regulations.

File: docs/changelog/changelog_en.md [hunk 3]
```diff
@@ -1,49 +1,46 @@
...
+- Improved the product title display for SEO purposes. Reverts to the default if no product specific title is available.
```
Comment (Tim-M-S) [inline docs/changelog/changelog_en.md]: This is actually a new field available on the items' data. This means you can either have a custom title or it is using the old pattern of "Item name | Company name"

File: docs/changelog/changelog_en.md [hunk 4]
```diff
@@ -1,49 +1,46 @@
...
-- In cases where the basket is empty during the checkout process, the system will now redirect to `/cart` and display a notification.
+- Implemented a new notification design.
 - Increased default notification timeout from 3 to 5 seconds.
+- The `height` and `width` attributes set in Terra UI are now used only for full-size images of an item.
+- Moved the generation of missing language files based on the environment's language configuration to the build script. The locale configuration is now based on the language files in the `lang` directory.
+- Started unifying SDK/API error handling. Errors now return keys that can be translated in the frontend.
+- Improved manufacturer visuals.
+- Updated manufacturer translation text.
+- Added support for a second argument to the payment status on an order. This allows you to define custom translations for different payment states.
+- If the cart is empty during the checkout process, the user is now redirected to the cart.
```
Comment (Tim-M-S) [inline docs/changelog/changelog_en.md]:
```suggestion
- If the cart is emptied during the checkout process, the user is now redirected to the cart.
```

File: docs/changelog/changelog_en.md [hunk 5]
```diff
@@ -1,49 +1,46 @@
...
+- The logo container is now more flexible and adapts to the dimensions of the provided logo.
```
Comment (Tim-M-S) [inline docs/changelog/changelog_en.md:44]:
```suggestion
- The logo container is now more flexible and adapts to the dimensions of the provided custom logos.
```

File: docs/changelog/changelog_en.md (hunk 1 above)
Comment (abocsan-plenty) [reply docs/changelog/changelog_en.md:22]: @Tim-M-S 
quick checkout is not controlled by environment setting, only the timer is.

Comment (ksted) [reply docs/changelog/changelog_en.md:22]: According to Andrei, no.

File: docs/changelog/changelog_en.md (hunk 5 above)
Comment (ksted) [reply docs/changelog/changelog_en.md:44]: I don't think "custom" adds any valuable information here. In the end all logos are custom logos, even the plenty one we use.

File: docs/changelog/changelog_en.md [hunk 6]
```diff
@@ -1,50 +1,47 @@
...
+- To ensure compliance with geo-blocking regulations, selecting an EU country as the shipping country now makes all EU countries available as billing options.
+- The product title display now supports custom titles. You can set the custom title on the item. If no custom title is set, the default "Item name | Company name" is used.
+- Added tax and shipping note to the wishlist page.
+- Added a new request header for the `configId`.
+- Added the environment variable `NO_CACHE` for disabling caching.
+- The hero banner and media card now support alt text properties for images.
+- Enabled font color customization for the hero banner via a template property.
+- Added a new carousel compoment.
+- Added an edit mode toolbar and JSON editor for the front end. Note that this is a preparatory step. Further functionality will be added in an upcoming version.
+- Added Google Pay and Apple Pay as new payment methods.
```
Comment (maxiroellplenty) [inline docs/changelog/changelog_en.md]:
```suggestion
- Google Pay and Apple Pay are now available as payment methods in the checkout.
```

File: docs/changelog/changelog_de.md [hunk 7]
```diff
@@ -1,5 +1,48 @@
 # Changelog plentyshopPWA
 
+## v1.7.0 (2024-11-06) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.6.0...v1.7.0" target="_blank" rel="noopener"><b>Übersicht aller Änderungen</b></a>
+
+### 🚀 Neu
+
+- Ist die Adresse beim Klicken auf **Kaufen** nicht gespeichert, wird jetzt eine Benachrichtigung angezeigt.
```
Comment (maxiroellplenty) [inline docs/changelog/changelog_de.md:8]:
```suggestion
- Ist die Adresse beim Klicken auf **Kaufen** nicht gespeichert, wird jetzt eine Benachrichtigung angezeigt.
- Google Pay und Apple Pay sind ab sofort als Zahlungsarten im Checkout verfügbar.
```

**Review by maxiroellplenty — APPROVED** [review]

**Pull request merged**